commit 80e93eecae52f2f90b6b86a0d283ef410ab59f75 (refs/changes/96/1196/2)
Author: Dillon Amburgey <dillona@dillona.com>
Date:   2017-01-03T21:25:59+00:00 (2 years, 9 months ago)
    
    joyent/triton#222 Support coal on CentOS 7 KVM
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Chris Burroughs <chris.burroughs@joyent.com>

diff --git a/tools/coal-linux-kvm-setup b/tools/coal-linux-kvm-setup
index b7ef212..350afc7 100755
--- a/tools/coal-linux-kvm-setup
+++ b/tools/coal-linux-kvm-setup
@@ -19,6 +19,8 @@
 #
 # This script is highly experimental and is community supported (it is not used by anyone at 
 # Joyent). This script has been tested on the following Linux distributions: 
+# * CentOS 7
+#   * Install required packages: yum install qemu-kvm libvirt virt-manager
 # * Fedora 23 
 #   * Install required packages: dnf install libvirt virt-manager
 # * Ubuntu 14.04, 16.04
@@ -63,6 +65,8 @@ if [ -e "/usr/bin/qemu-kvm" ]; then
     EMULATOR=/usr/bin/qemu-kvm
 elif [ -e "/usr/bin/kvm" ]; then
     EMULATOR=/usr/bin/kvm
+elif [ -e "/usr/libexec/qemu-kvm" ]; then
+    EMULATOR=/usr/libexec/qemu-kvm
 else
     echo "KVM cannot be found. Exiting."
     exit 1
